app/src/ui/app.tsx

```diff
@@ -96,7 +96,7 @@ export class App extends React.Component<IAppProps, IAppState> {
       this.forceUpdate()
 
       requestIdleCallback(() => {
-        const now = Date.now()
+        const now = performance.now()
```

Comment:
Oh, maybe it's the other way around and `window.performance` isn't available in the main process.

Anyway, a little shout out to this difference would be :gem: